packages/shared/src/components/filters/MyFeedHeading.tsx

```diff
@@ -73,7 +61,7 @@ function MyFeedHeading({
 
   const alertProps: Omit<AlertPointerProps, 'children'> = {
     offset: getOffset(),
-    isAlertDisabled: shouldHighlightFeedSettings ? isAlertDisabled : true,
+    isAlertDisabled: true,
```

Comment:
This means there will never be an alert, should we remove it completely instead?

Reply:
Ah yes, wasn't sure what the alert was here, but can check and do that yes.